### PR TITLE
Fixes lint issues

### LINKED ISSUE #1018: fixing lint issues
## Bug Report

Fix all the lint issues and ensure that CI fails for future PRs creating new lint issues.

Check the lint issues in [this action run](https://github.com/kubearmor/KubeArmor/actions/runs/3691275515).

**General Information**

- Environment description (GKE, VM-Kubeadm, vagrant-dev-env, minikube, microk8s, ...)
all
- Kernel version (run `uname -a`)
all

Tasks:
- [ ] Fix existing lint issues
- [ ] Fail the CI if a new lint issue appears.

**To Reproduce**

1. Run `make golint`
2. After raising the PR, check for any lint issues reported in the action/

**Expected behavior**
1. No lint issues are reported.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by vishal-chdhry**
Signed-off-by: Vishal Choudhary <contactvishaltech@gmail.com>

**Purpose of PR?**:
Fixes lining issues generated while running `make golint`
Fixes #1018 

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [x] Bug fix. Fixes #1018 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

**Comment by vishal-chdhry** [comment]
@daemon1024 @nyrahul Made some changes, you might want to take a look.
And I am not entirely aware of what every package does so i couldn't write a descriptive comment about it.

**Comment by nyrahul** [comment]
I am mostly good to merge these changes. Can you please fix the merge conflicts?

**Comment by vishal-chdhry** [comment]
@nyrahul done!

File: KubeArmor/common/common.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package common declaration
```
Comment (daemon1024) [inline KubeArmor/common/common.go]:
```suggestion
// Package common contains utility functions which are commonly used across packages and modules
```

File: KubeArmor/core/containerdHandler.go [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package core declaration
```
Comment (daemon1024) [inline KubeArmor/core/containerdHandler.go]:
```suggestion
// Package core is responsible for initiating and maintaining interactions between external entities like K8s,CRIs and internal KubeArmor entities like eBPF Monitor and Log Feeders
```

Let's move this comment declaration to `core/kubeArmor.go`

File: KubeArmor/enforcer/appArmorEnforcer.go [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package enforcer declaration
```
Comment (daemon1024) [inline KubeArmor/enforcer/appArmorEnforcer.go]:
```suggestion
// Package enforcer is responsible for setting up and handling policy updates for supported enforcers including AppArmor, SELinux and BPFLSM
```

Let's move this comment to `enforcer/runtimeEnforcer.go`

File: KubeArmor/enforcer/SELinuxEnforcer.go [hunk 4]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package enforcer declaration
```
Comment (daemon1024) [inline KubeArmor/enforcer/SELinuxEnforcer.go]:
```suggestion
```

duplicate

File: KubeArmor/feeder/feeder.go [hunk 5]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package feeder declaration
```
Comment (daemon1024) [inline KubeArmor/feeder/feeder.go]:
```suggestion
// Package feeder is responsible for sanitizing and pushing telemetry data to connected clients 
```

File: KubeArmor/kvmAgent/kvmAgent.go [hunk 6]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package kvmagent declaration
```
Comment (daemon1024) [inline KubeArmor/kvmAgent/kvmAgent.go]:
```suggestion
// Package kvmagent contains utilities to connect to kvmservice to establish support on bare-metal env
```

File: KubeArmor/log/logger.go [hunk 7]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package log declaration
```
Comment (daemon1024) [inline KubeArmor/log/logger.go]:
```suggestion
// Package log contains log util wrappers for enhanced pretty logging
```

File: KubeArmor/main.go [hunk 8]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package main declaration
```
Comment (daemon1024) [inline KubeArmor/main.go]:
```suggestion
// Package main is the entrypoint to initializing the armor
```

File: KubeArmor/policy/policy.go [hunk 9]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package policy declaration
```
Comment (daemon1024) [inline KubeArmor/policy/policy.go]:
```suggestion
// Package policy handles policy updates over gRPC in non-k8s environment
```

**Review by daemon1024 — CHANGES_REQUESTED** [review]
Hey @Vishal-Chdhry, I have added some suggestions and expanded on the comments.

**Review by daemon1024 — APPROVED** [review]

**Review by nyrahul — APPROVED** [review]
LGTM

**Pull request merged**